subcommands/targets/prune.go

```diff
@@ -26,6 +26,21 @@ func init() {
 		Short: "Prune target(s)",
 		Run:   doPrune,
 		Args:  cobra.MinimumNArgs(1),
+		Example: `
+  # prune a single target name its name:
+  fioctl targets prune intel-corei7-64-lmp-123
+
+  # prune all targets with the "custom.tags" set to ["devel"]:
+  fioctl targets prune --by-tag devel
+
+  # prune all targets with the "custom.tags" set to ["devel"] except for the last 10:
```

Comment:
I like "last 10" but i wonder if "most recent 10" would make people more comfortable? I don't have a strong opinion.

Reply:
Changed in force-push.